CMakeLists.txt

```diff
@@ -34,15 +34,31 @@ set(depot_tools_install_dir ${SOURCE_DIR})
 
 set(WEBRTC_REVISION branch-heads/70)
 
-list(APPEND GN_GEN_ARGS
-  rtc_build_examples=false
-  rtc_build_tools=false
-  rtc_use_x11=false
-  rtc_enable_protobuf=false
-  rtc_use_gtk=false
-  rtc_include_pulse_audio=false
-  rtc_include_tests=false
-)
+if ("$ENV{TARGET_ARCH}" STREQUAL "arm")
+  list(APPEND GN_GEN_ARGS
+    target_os="linux"
+    target_cpu="arm"
+    rtc_build_examples=false
+    rtc_build_tools=true
+    rtc_use_x11=false
+    rtc_enable_protobuf=false
+    rtc_use_gtk=false
+    rtc_include_pulse_audio=false
+    rtc_include_tests=false
+    treat_warnings_as_errors=false
+    fatal_linker_warnings=false
+  )
+else()
+    list(APPEND GN_GEN_ARGS
+      rtc_build_examples=false
+      rtc_build_tools=false
+      rtc_use_x11=false
+      rtc_enable_protobuf=false
+      rtc_use_gtk=false
+      rtc_include_pulse_audio=false
+      rtc_include_tests=false
+  )
```

Comment:
Can you factor out the common args? For example,

* `rtc_build_examples=false`
* `rtc_use_x11=false`
* `rtc_enable_protobuf=false`
* `rtc_use_gtk=false`
* `rtc_include_pulse_audio=false`
* `rtc_include_tests=false`

Reply:
OK, it could be better.
I will make it.

Reply:
Ok it is not so straight, I need help for travis.
Could you please help it?
Here is necessary commands for cross-compiling.
```
wget https://releases.linaro.org/components/toolchain/binaries/latest-7/arm-linux-gnueabihf/gcc-linaro-7.3.1-2018.05-i686_arm-linux-gnueabihf.tar.xz
tar xf gcc-linaro-7.3.1-2018.05-i686_arm-linux-gnueabihf.tar.xz
SKIP_DOWNLOAD=true TARGET_ARCH=arm ARM_TOOLS_PATH=$(pwd)/gcc-linaro-7.3.1-2018.05-i686_arm-linux-gnueabihf npm install
```